docs/server/source/cloud-deployment-templates/server-tls-certificate.rst

```diff
@@ -1,86 +1,28 @@
-Generate Cluster Member Certificate for MongoDB
-===============================================
+How to Generate a Server Certificate for MongoDB
+================================================
 
```

Comment:
There are 2 ways about it:
MongoDB calls it a `cluster member` certificate, and usual PKI terminology is `server` certificate.
I think we can stick with the PKI one to be agnostic of MongoDB terminology.